lib/fathead/arrow_docs/parse.py

```diff
@@ -37,8 +37,11 @@ def yield_api_example(api_examples):
 
         abstract = clean_output(api_example.dd.p.text)
         abstract = abstract_fmt.format(abstract, code)
+        
+        # redirect_alias: => 'arrow.arrow.Arrow.clone' => arrow clone
+        redirect_alias = 'arrow ' + title.split('.')[-1]
 
-        yield [title, 'A', '', '', '', '', '', '', '', '', '', abstract, url]
```

Comment:
@danidee10 you need to keep this line. You want to _add_ redirects to what we have. Not replace what we have with only redirects.